### PR TITLE
A few more more Tox cleanup bits

### PR DESCRIPTION
**Pull request by wsanchez**
While merging #106 forward, I found a few more bits to clean up.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -265,7 +265,7 @@ commands =
     sphinx-build                           \
         -b html -d "{envtmpdir}/doctrees"  \
         "{toxinidir}/docs"                 \
-        "{toxworkdir}/docs/html"
```
Comment (wsanchez) [inline tox.ini:268]: This is to match other projects, and similar to `/htmlcov/`.

Comment (mahmoud) [reply tox.ini:268]: Just curious, other hyper projects, or which other projects?

Comment (wsanchez) [reply tox.ini:268]: Klein in particular; I'm trying to keep the configs relatively similar for sanity.

Comment (glyph) [reply tox.ini:268]: /me whispers "we need a tool to keep them in sync"

Comment (wsanchez) [reply tox.ini:268]: I KNOW

**Pull request merged**